### PR TITLE
chore: add codeowners and repo settings

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

**Review by miraleung — CHANGES_REQUESTED** [review]
Can we make the owners settings narrower for this repo? Since it's in development, let's start with myself, and we can add to that set as more folks become familiar with this codebase.

**Comment by JustinBeckwith** [comment]
Sure thing!  For which part?  
- For CODEOWNERS right now, actools or yoshi-java can sign off - do you want that to just be you?
- For write/admin access, it's set up kind of standard java.  Do you want to change that bit?

**Comment by summer-ji-eng** [comment]
> Sure thing! For which part?
> 
> * For CODEOWNERS right now, actools or yoshi-java can sign off - do you want that to just be you?
> * For write/admin access, it's set up kind of standard java.  Do you want to change that bit?

Currently, @miraleung, @summer-ji-eng @xiaozhenliu-gg5 are the main contributors to this repo.

**Comment by JustinBeckwith** [comment]
I just yanked any team config to reduce confusion.  Feel free to update if/when you like!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+*     @miraleung, @summer-ji-eng @xiaozhenliu-gg5
```
Comment (summer-ji-eng) [inline .github/CODEOWNERS:7]: No need comma here.

**Comment by summer-ji-eng** [comment]
Thanks @JustinBeckwith helping us on this. LGTM, wait for @miraleung take another look.

**Comment by JustinBeckwith** [comment]
Thanks!  Ready for another look @miraleung :)

**Pull request merged**